app/frontend/state.ts

```diff
@@ -285,4 +286,6 @@ export const getActiveAccountInfo = (state: State) => {
   assert(state.accountsInfo[state.sourceAccountIndex] != null)
   return state.accountsInfo[state.activeAccountIndex]
 }
+export const getChangeAddress = (accountInfo: AccountInfo): Address =>
```

Comment:
I'd put this helper rather in `app/frontend/wallet/account.ts` file, here I find it a bit misplaced as I see the "state" as something higher-level